### ISSUE TITLE
Settings File Address Resolution

### ISSUE THREAD

**Issue by fiaranch** [evilbunny2008/weeWxWeatherApp#26]
I looked at all issues here and searched the WeeWX google group and did not see a resolution.  Please point me in the right direction if I missed it.

CURRENT STATE: Running WeeWX v4.9.1 and using Rsync to upload data to my domain public server.  All that works fine.
Installed the Inigo skin, set up the settings file, it is uploaded to the public server.
The data file is uploading with the Rsync job (every 2 minutes)
Installed the App from Google Play on Samsung Note Smartphone.  Also installed it on a Samsung Tablet.
Launch the app, enter the path to the settings file and it fails.

ERROR MESSAGE:  Wasn't able to connect or download data.txt from your server.
java.net.UnknownHostException: Unable to resolve host "fiaranch" (my domain): No address associated with host name.

Both files (settings and data) come up when requested in a browser on either device using either http or https:
Paths are:
https://fiaranch.com/weather/weewx/inigo-settings.txt
https://fiaranch.com/weather/weewx/inigo-data.txt

Researching the error, it keeps coming back to some user authorization issue in some manifest.

Again, if I missed the resolution for this issue, please point me in the right direction.

Thanks!!

**Comment by evilbunny2008** [comment]
I get a bluehost banner on both URLs

**Comment by fiaranch** [comment]
Sorry.

 

Because I did not hear back in such a long time, I removed the skin from WeeWX, deleted the directory on my site,  and went another direction, hence the blue host failure.

 

If you have interest in chasing the bug, when I have some time I can reinstall the skin, and set it all up again.

 

Regards

 

Dwight

**Comment by evilbunny2008** [comment]
I'm not sure what's happening, I'm not getting notifications from github, as for your problem, it looked like a DNS issue

**Comment by fiaranch** [comment]
I was on the run yesterday, so let me be a little more clear this morning.

First, to your thought, no, it is not a DNS issue.  As I said in my first email, I am not a coder, but when I did some cursory research on the error, it points to some manifest in the app that is not telling android that it has internet access permission.

So this morning I went and looked and when I rearranged my site, the two inigo files are still sitting there in the weewx directory.  So I reinstalled the app on my phone, to to make sure it was not some path problem originally, and got the same failure.  I did not reload the app on my tablet.

I uploaded a couple of screenshots of the failure.  I tried it both with https and http

The initial input screen shows the path input - http://fiaranch.com/image.store/weather/input.jpg

This is the error reported when touch "save settings" - http://fiaranch.com/image.store/weather/error.jpg

Here are the links to the existing inigo files:

Settings File - https://weather.fiaranch.com/weewx/inigo-settings.txt

Data File - https://weather.fiaranch.com/weewx/inigo-data.txt
NOTE: Since I removed the skin, I have no idea if this data file is current.

As I said, I decided to go a different direction, and found a skin that scales nicely from smartphone to tablet to desktop, so I ended up with this:  weather.fiaranch.com

If you have further questions, or would like to chase the bug, I will help however I can.

Regards,
Dwight



-------- Original Message --------
